doc/api/console.markdown

```diff
@@ -32,7 +32,7 @@ In daily use, the blocking/non-blocking dichotomy is not something you
 should worry about unless you log huge amounts of data.
 
 
-### console.log([data][, ...])
+### console.log(object [, object, ...])
```

Comment:
I've tested this before, Node and Firefox print an empty line, Chrome does not. Maybe it should be brought up with the spec authors?